### PR TITLE
Add option to disable adding logo to plots

### LINKED ISSUE #308: Make logo optional
We need to make the logo optional so people can choose to turn this off especially for publication plots. All examples need to default to this on though.

### PR DESCRIPTION
**Pull request by zmoon**
```yaml
analysis:
  add_logo: false
```

Resolves #308

### CODE REVIEW

**Comment by zmoon** [comment]
I tested with idealized example and it worked, but maybe someone could test a more complicated example just to make sure.

File: melodies_monet/driver.py [hunk 1]
```diff
@@ -738,6 +738,8 @@ def __init__(self):
         self.obs_gridded_data = {}
         self.obs_gridded_count = {}
         self.obs_gridded_dataset = None
+        self.add_logo = True
+        """bool, default=True : Add the MELODIES MONET logo to the output plots."""
```
Comment (zmoon) [inline melodies_monet/driver.py:742]:
```suggestion
        """bool, default=True : Add the MELODIES MONET logo to the plots."""
```

**Comment by mbruckner-work** [comment]
Tested with a satellite test case. The savefig calls within some of the satellite/xarray plotting routines currently include `decorate = True` as a hardwired argument. 
When these are removed, `add_logo` works as intended.

**Comment by zmoon** [comment]
> Tested with a satellite test case. The savefig calls within some of the satellite/xarray plotting routines currently include `decorate = True` as a hardwired argument. When these are removed, `add_logo` works as intended.

Thanks @mbruckner-work for pointing that out, I will address.

**Review by mbruckner-work — APPROVED** [review]
Looks good! 
From my review of the driver and the plotting routine sets, satplots was the only one with hardwired `decorate`.

**Pull request merged**